### PR TITLE
feat: add portfolio page

### LINKED ISSUE #119: feat(next): create portfolio page and route

### PR DESCRIPTION
**Pull request by 120EE0692**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #119 

### Description
Created portfolio page for the content team.

image

ready to deploy
image

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/components/portfolio/Backlink.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 // Libraries
 import { Container, makeStyles, Typography } from '@material-ui/core';
-import { Link } from 'react-router-dom';
 
 const BackLink = () => {
   const classes = useStyles();
   return (
     <Container>
       <div className={classes.wrapper}>
-        <Link to='/about' className={classes.link}>
```
Comment (rutajdash) [inline client/src/components/portfolio/Backlink.js:12]: Wasn't this fixed before?

File: client/src/components/shared/MarkdownWrapper.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ function MarkdownWrapper({ children, variant, styles }) {
 
   return (
     <ReactMarkdown
+      ordered='true'
```
Comment (rutajdash) [inline client/src/components/shared/MarkdownWrapper.js:12]: What is this for?

File: client/src/pages/portfolio/[userId]/[name].jsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+const PortfolioPage = ({ userId, firstName, lastName, email, profileImageLink, articleList, noOfArticle, year }) => {
+    const { isFallback } = useRouter();
+
+    const user = { userId, firstName, lastName, email, profileImageLink, articleList, noOfArticle, year };
+
+    return (
+        <>
+            <Marginals>
+
+                <UserContext.Provider value={user}>
```
Comment (rutajdash) [inline client/src/pages/portfolio/[userId]/[name].jsx:27]: Why do we need context here?

**Review by rutajdash — COMMENTED** [review]
The files are not formatted properly

File: client/src/pages/portfolio/[userId]/[name].jsx (hunk 3 above)
Comment (120EE0692) [reply client/src/pages/portfolio/[userId]/[name].jsx:27]: The user card is deeply nested.

File: client/src/components/portfolio/Backlink.js (hunk 1 above)
Comment (120EE0692) [reply client/src/components/portfolio/Backlink.js:12]: No

**Comment by 120EE0692** [comment]
> 

All files are formated on my system.

File: client/src/pages/portfolio/[userId]/[name].jsx (hunk 3 above)
Comment (rutajdash) [reply client/src/pages/portfolio/[userId]/[name].jsx:27]: You will simply need to pass the user data to Portfolio.jsx and then to UserCard.jsx .
This is not too much.

File: client/src/components/shared/MarkdownWrapper.js (hunk 2 above)
Comment (rutajdash) [reply client/src/components/shared/MarkdownWrapper.js:12]: Why is JSON being passed into ReactMarkdown?

**Review by rutajdash — APPROVED** [review]

**Pull request merged**